controllers/routes/userRoutes.js

```diff
@@ -0,0 +1,77 @@
+const router = require('express').Router();
+const { User } = require('../models');
+const withAuth = require('../utils/auth');
+
+// Get user profile
```

Comment:
Lines 6 through 21: I would make this a low priority item. Pulling up user profiles is a good feature, but it may be something we have to forego due to time limits.